liquibase-standard/src/main/java/liquibase/executor/jvm/JdbcExecutor.java

```diff
@@ -188,7 +188,28 @@ public void execute(final SqlStatement sql, final List<SqlVisitor> sqlVisitors)
     private void setParameters(final PreparedStatement pstmt, final RawParameterizedSqlStatement sql) throws SQLException {
         final List<Object> parameters = sql.getParameters();
         for (int i = 0; i < parameters.size(); i++) {
-            pstmt.setObject(i + 1, parameters.get(i));
+            Object parameter = parameters.get(i);
```

Comment:
Seems we are bringing some commits from master here

Reply:
Good catch